web/themes/custom/drupalam/scss/component/_signin-signup.scss

```diff
@@ -21,6 +21,11 @@
       }
     }
   }
+  .block-join-now-block {
+    .contextual {
```

Comment:
Double-check this please as an anonymous user.

Reply:
These CSS styles only for Editors/Admins, some contextual links hard are not accessible and I moved them to be clickable.